app/components/containers/containersController.js

```diff
@@ -8,6 +8,7 @@ angular.module('containers', [])
   $scope.sortType = 'State';
   $scope.sortReverse = false;
   $scope.state.selectedItemCount = 0;
+  $scope.state.selectedItemRunning = 0;
```

Comment:
I do not like the fact of managing the number of running items in the state. I'd rather iterate through the list of selected containers when clicking on the remove action to see if there is at least a running container and then display the modal box.